app/controllers/application_controller.rb

```diff
@@ -14,41 +14,30 @@ def invalid_request(message = error_msg, status = 400)
     render json: {errors: message}, status: status
   end
 
-  def login
-    user = authenticate_cookie
-    @current_user = User.from_andela_auth(user) if user && !user['isGuest']
-    return unauthorized_token unless @current_user
-
-    @token = TokenManager.generate_token(@current_user.id)
-  end
-
-  def logout
-    cookie = request.headers['HTTP_ANDELA_COOKIE']
-    return invalid_request("Cookie must be provided") unless cookie
-    CookieHandler.logout_cookie(cookie)
-    render json: {message: 'logged out'}, status: 200
-  end
+   def login
```

Comment:
not sure we'd be needing this route! Also, it creates a source for a bug on the current user.
*What do I mean?*
Imagine you've assigned the `@current_user` in `authenticate_user` and then this is reassigning the value, however at this point, `authenticate_user` might return a `@current_user` or an `unauthorized_token` response. Technically, the `unauthorized_token` should terminate the execution flow, however we want to ensure that only a `User` object is stored as the `current_user`, not just anything! Unless you want to end up checking whether the current_user is a User all over your application. Not sure of why we still need the route though, since auth is being done on the FE!